charts/rancher-monitoring/v0.0.4/values.yaml

```diff
@@ -164,6 +164,21 @@ exporter-node:
   ##
   serviceAccountName: ""
 
+exporter-node-windows:
+  enabled: false
+  apiGroup: "monitoring.coreos.com"
+  endpoints: []
+  nodeSelectors: []
+  image:
+    repository: micheliac/wmi-exporter-package
+    tag: windows-1809
+  ports:
```

Comment:
tag should be something like `v0.0.1`

Reply:
Please change repository to `rancher/wmi_exporter-package` and tag `v0.0.1`

https://hub.docker.com/r/rancher/wmi_exporter-package/tags